### PR TITLE
[AUTO] Set additional property into meta plugin for compiling stateful LLM

### PR DESCRIPTION
**Pull request by yangwang201911**
Should to set additional property to meta plugin when compiling stateful LLM

ticket: CVS-154229

### CODE REVIEW

File: src/cpp/src/llm_pipeline.cpp [hunk 1]
```diff
@@ -77,19 +78,26 @@ class StatefulLLMPipeline final : public LLMPipelineImplBase {
...
+            meta_plugin_config.emplace(ov::intel_auto::enable_startup_fallback(false));
+            meta_plugin_config.emplace(ov::intel_auto::enable_runtime_fallback(false));
+        }
         if(auto filtered_plugin_config = extract_adapters_from_properties(plugin_config, &m_generation_config.adapters)) {
             auto [core_plugin_config, compile_plugin_config] = ov::genai::utils::split_core_complile_config(*filtered_plugin_config);
             core.set_property(core_plugin_config);
             auto model = core.read_model(model_path / "openvino_model.xml");
             m_adapter_controller = AdapterController(model, m_generation_config.adapters, "base_model.model.model.", device);   // TODO: Make the prefix name configurable
             utils::slice_matmul_statefull_model(model);
+            compile_plugin_config.insert(meta_plugin_config.begin(), meta_plugin_config.end());
```
Comment (peterchen-intel) [inline src/cpp/src/llm_pipeline.cpp:92]: Need to check if any of these 2 has already been in the config.

Comment (yangwang201911) [reply src/cpp/src/llm_pipeline.cpp:96]: Updated.

**Comment by ilya-lavrenov** [comment]
Looks like the same should be enabled in VLM pipeline for LLM part

**Comment by peterchen-intel** [comment]
This issue doesn't reproduce since genai has been changed to pass ov::model instead of passing model path.
Passing model path to AUTO will be handled in AUTO

**Pull request closed without merging**